src/gfx/render.js

```diff
@@ -87,6 +87,19 @@ var ARROW_POINTS = [
  -10.0 * R,  0.9 * R2, 0
 ];
 
+// van der Waals radius by atom._element
+var VDW_RADIUS = {
+  H: 1.2,
+  C: 1.7,
+  N: 1.55,
+  O: 1.52,
+  F: 1.47,
+  P: 1.8,
+  S: 1.8,
+  CL: 1.75,
+  CU: 1.4
+};
```

Comment:
This map is missing many of the elements. It would be good to at least have all the elements in CPK_TABLE (src/colors.js).